service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -125,20 +143,56 @@ void shouldReturnErrorsWhenFamilymanNumberIsNotProvided() {
         assertThat(callbackResponse.getErrors()).containsExactly("Enter Familyman case number");
     }
 
-    @Test
-    void shouldTriggerOrderEventWhenSubmitted() throws Exception {
-        String expectedCaseReference = "19898989";
-        postSubmittedEvent(buildCallbackRequest());
+    @TestInstance(PER_CLASS)
+    @Nested
+    class Submitted {
+
+        @AfterEach
+        void resetInvocations() {
+            reset(notificationClient);
+        }
+
+        @Test
+        void submittedShouldNotifyAdminAndLAWhenNoRepresentativesNeedServing() throws Exception {
+            postSubmittedEvent(buildCallbackRequest());
+
+            verify(notificationClient).sendEmail(
+                eq(ORDER_NOTIFICATION_TEMPLATE_FOR_LA), eq(LOCAL_AUTHORITY_EMAIL_ADDRESS),
+                eq(expectedOrderLocalAuthorityParameters()), eq(CASE_ID));
+
+            verify(notificationClient).sendEmail(
+                eq(ORDER_NOTIFICATION_TEMPLATE_FOR_ADMIN), eq("admin@family-court.com"),
+                eq(getExpectedParametersForAdminWhenNoRepresentativesServedByPost()), eq(CASE_ID));
+
+            verifyZeroInteractions(notificationClient);
+        }
+
+        @Test
+        void submittedShouldNotifyAdminAndLAWhenRepresentativesNeedServingByPost() throws Exception {
+            given(documentDownloadService.downloadDocument(anyString(), anyString(), anyString()))
+                .willReturn(PDF);
+
+            postSubmittedEvent(buildCallbackRequestWithRepresentatives());
+
+            verify(notificationClient).sendEmail(
+                eq(ORDER_NOTIFICATION_TEMPLATE_FOR_LA), eq(LOCAL_AUTHORITY_EMAIL_ADDRESS),
+                eq(expectedOrderLocalAuthorityParameters()), eq(CASE_ID));
+
+            verify(notificationClient).sendEmail(
+                eq(ORDER_NOTIFICATION_TEMPLATE_FOR_ADMIN), eq("admin@family-court.com"),
+                dataCaptor.capture(), eq(CASE_ID));
+
+            MapDifference<String, Object> difference = verifyNotificationSentToAdminWhenOrderIssued(dataCaptor);
+            assertThat(difference.areEqual()).isTrue();
 
-        verify(notificationClient).sendEmail(
-            eq(ORDER_NOTIFICATION_TEMPLATE), eq(LOCAL_AUTHORITY_EMAIL_ADDRESS),
-            eq(expectedOrderLocalAuthorityParameters()), eq(expectedCaseReference));
+            verifyZeroInteractions(notificationClient);
+        }
     }
 
     private CallbackRequest buildCallbackRequest() {
         return CallbackRequest.builder()
             .caseDetails(CaseDetails.builder()
-                .id(19898989L)
+                .id(12345L)
```

Comment:
why change?

Reply:
for my tests I'm expecting case ID to be 12345 so I need to change the ID here otherwise my tests will fail, hope it makes sense

Reply:
you use CASE_ID in some places, but then not here. Please be consistent